src/ui/component/stepper/stepperProgress/progressBar/progressBar.tsx

```diff
@@ -0,0 +1,25 @@
+import classNames from 'classnames'
+import './progressBar.scss'
+
+export type ProgressBarColor = 'grey' | 'blue' | 'green'
+
+export type ProgressBarTransition = 'none' | 'backward' | 'forward'
+
+type ProgressBarProps = {
+  transition?: ProgressBarTransition
+  color?: ProgressBarColor
+}
+
+export const ProgressBar: React.FC<ProgressBarProps> = ({
+  color = 'grey',
+  transition = 'none'
+}) => {
+  return (
+    <div
+      className={classNames('okp4-dataverse-portal-progress-bar-main', color, transition)}
+      style={{
+        transitionDelay: transition === 'forward' ? '600ms' : '0ms' // 600ms must match the transitions and animations durations in progressBar.scss file
```

Comment:
Isn't it easier to add `transition-delay: 600ms;` in `forward` class scss?

Reply:
The transition delay is related to the `background-color` transition for the progress bar element, and doesn't affect the pseudo element animation. For more clarity, we might define a dedicated class instead of inlining the property?

Reply:
thanks for your explanation, yes I think we should avoid inline style to remain as consistant as possible 👍